webviz_subsurface/plugins/_structural_uncertainty/structural_uncertainty.py

```diff
@@ -188,28 +186,30 @@ def __init__(
             ens: webviz_settings.shared_settings["scratch_ensembles"][ens]
             for ens in ensembles
         }
+
+        # Create a table of surface files
         surface_table = find_surfaces(self._ensemble_paths)
+        # Filter on provided surface attributes
         surface_table = surface_table[surface_table["attribute"].isin(self._surf_attrs)]
+        # Filter on provided surface names
+        self._surfacenames = (
```

Comment:
Question, I have not tested yet.. but what happens if the user has a typo in one of the names and it does not exist, will the plugin handle that? should we check against the available names and warn if the user has given an invalid name?